examples/tcp-listener/tcp_listener.bal

```diff
@@ -0,0 +1,44 @@
+// This is the server implementation for the TCP socket.
+import ballerina/io;
+import ballerina/log;
+import ballerina/tcp;
+
+// Bind the service to the port. 
+service on new tcp:Listener(3000) {
+
+    // This remote method is invoked when the new client joins the server.
+    remote function onConnect(tcp:Caller caller) returns tcp:ConnectionService {
+        io:println("Client connected to echoServer: ", caller.remotePort);
```

Comment:
Shall we print the host/IP which connected here as well. IP/port combination would be more meaningful here.

Reply:
+1